cli/cmd/aeon.go

```diff
@@ -16,27 +17,45 @@ import (
 	"github.com/tarantool/tt/cli/modules"
 	"github.com/tarantool/tt/cli/running"
 	"github.com/tarantool/tt/cli/util"
-	"github.com/tarantool/tt/lib/cluster"
+	libcluster "github.com/tarantool/tt/lib/cluster"
 	libconnect "github.com/tarantool/tt/lib/connect"
+	"github.com/tarantool/tt/lib/integrity"
 )
 
 const (
 	aeonHistoryFileName = ".aeon_history"
 	aeonHistoryLines    = console.DefaultHistoryLines
 )
 
+var aoenHelp = libconnect.MakeURLHelp(map[string]any{
+	"https":     "connection protocol",
+	"user:pass": "authentication",
+	"localhost": "host (server address)",
+	":2379":     "port, Default client port (the distributed key-value store)",
+	"/prefix": "key prefix (optional)," +  
```

Comment:
Please, see golangci-lint linter errors and fix it.

Reply:
Done